scripts/amplience/clean.ts

```diff
@@ -99,8 +114,25 @@ export const cleanHandler = async (context: Arguments<Context>): Promise<any> =>
     }))
 
     console.log('Deleting mapFile...')
-    rmSync(mappingFile, {force: true})
+    rmSync(mappingFile, { force: true })
 
-    console.log(`Done!`)
+    // novadev-581 Update automation so that cleanup removes all folders from repositories
```

Comment:
Weird to have this as a comment.

Reply:
We can clean these comments as the branches are removed anyway